app.js

```diff
@@ -247,6 +247,28 @@ app.post('/inventory/remove_item', function (req, res) {
     })
 })
 
+// Inventory - Modify Item Route
+app.post('/inventory/modify_item', function(req, res) {
+    
+    // Grab the necessary data from the POST request body
+    var id = req.body.modal_shelf_id;
+    var min = req.body.modal_shelf_minimum_threshold
+    var max = req.body.modal_shelf_maximum_threshold
```

Comment:
```suggestion
    const id = req.body.modal_shelf_id;
    const min = req.body.modal_shelf_minimum_threshold
    const max = req.body.modal_shelf_maximum_threshold
```
If @quinn-codes-synthesis does warehouse min/max thresholds and uses the same variable names...then would we get in trouble declaring with `var`?